diktat-rules/src/test/resources/test/paragraph2/kdoc/package/src/main/kotlin/com/saveourtool/diktat/kdoc/methods/KdocWithoutThrowsTagTested.kt

```diff
@@ -0,0 +1,13 @@
+package test.paragraph2.kdoc.`package`.src.main.kotlin.com.saveourtool.diktat.kdoc.methods
```

Comment:
I think this file should be end with `Test` and not `Tested`

Reply:
Discussed offline